### DISCUSSION TITLE
domain used by others

### DISCUSSION THREAD

**Discussion by xingpingcn** [vercel/community#6285] [Help] [answered]
### Summary

hi, i have 2 domains, one of them is xingpingcn.top, one is xingpingcn.eu.org, when i add those subdomain to my project, i was told that the domains has been used by other account, which stops me adding subdomain via API. howerver, i can add many subdomains(both of the 2 apex domain) via api with on limitation just a few days ago. so is there any solution that i can add subdomain without any TXT record and just verify once that I am the owner

### Steps to Reproduce

add subdomain via dash

**Comment by amyegan** [comment]
Hey @xingpingcn. You can verify that you control that domain by adding the requested TXT record. You can learn more about how it works at https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access

**Reply by xingpingcn** [reply]
Thanks Reply. 

It would be better if you could tell me which account is using my domain name, because before this few days I have never had to add a TXT record to verify that this is my domain. 

In addition, I cannot use wildcard domain. When I click enable vercel dns, it will prompt "*.iptest.xingpingcn.top" does not exist(i know i have to add NS records, but it doesnt prompt me what kind of NS records should be added). So if I can only use TXT record verification, I have to verify it every time I add a subdomain, which will become very troublesome when I need to add a lot of subdomains at the same time
![image](https://github.com/vercel/community/assets/117259690/7b04790f-000f-47f9-8dfa-77e60132152e)

**Reply by amyegan** [reply, marked as answer]
That's not what I would normally expect to happen. 🤔  I think your account must have been updated to our new structure in the last few days. If that's the case, your domain may need to be moved from your personal account to your new hobby team. You can learn more about that in https://github.com/vercel/community/discussions/6298

**Reply by xingpingcn** [reply]
Thanks for your new created announcement! This is totally tailor-made for me

**Comment by hadescharon** [comment]
Another Vercel account is using this domain. Set the following TXT record on _vercel.cloudns.be to use liziyu.cloudns.be in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.”I should send TXT using the domain name_vers.cloudns.be, but my service provider can only send TXT using the domain name_vers.liziyu.cloudns.be. Will my verification pass?

**Reply by xingpingcn** [reply]
free subdomain dont work mate
